gemini/use-cases/retrieval-augmented-generation/qna_using_query_routing/qna_using_query_routing.ipynb

```diff
@@ -0,0 +1,1008 @@
+{
```

Comment:
<p>Pandas should be able to read/write a JSONL file directly from GCS, and this code could be simplified. Just put the gs:// path for as the filepath</p><p> </p><p><a href="https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html#pandas-read-csv" target="_blank">https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html#pandas-read-csv</a></p><p> </p><p>(I haven't done this in a while, so It's possible I'm mistaken)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/493/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>